addons/web_editor/static/src/js/editor/snippets.editor.js

```diff
@@ -3015,7 +3016,8 @@ var SnippetsMenu = Widget.extend({
             !$(range.commonAncestorContainer).parents('#wrapwrap, .iframe-editor-wrapper .o_editable').length ||
             $(range.commonAncestorContainer).parent('[data-oe-model]:not([data-oe-type="html"]):not([data-oe-field="arch"])').length ||
             (e && $(e.target).closest('.fa, img').length ||
-            this.options.wysiwyg.lastMediaClicked && $(this.options.wysiwyg.lastMediaClicked).is('.fa, img'))
+            this.options.wysiwyg.lastMediaClicked && $(this.options.wysiwyg.lastMediaClicked).is('.fa, img')) ||
+            !this.options.wysiwyg.lastElement.isContentEditable
```

Comment:
Seems to be working but I am not sure to understand why, in any case, fine by me but can you confirm with the editor team? Or did you already maybe? If not, ping AGE here for example.

Reply:
As the spec of the task was 
> no inline text on field selection: https://i.imgur.com/awrHQx1.png

I thought the behaviour (remove toolbar) should be applied for every content that is not editable. To do that, we test on the last element clicked at the wysiwyg level. @Zinston should I add you as a reviewer, or RD-Editor ?

Reply:
Our reviewer-in-chief is rather @dmo-odoo but about this line, the approach makes sense to me if indeed what you want is no toolbar on non-editable content. However be careful as `lastElement` will not always be defined so you might get `undefined` errors here. This is because you defined `lastElement` as `$target[0]` where `$target = e ? editorWindow.$(e.target) : editorWindow.$()`.

Reply:
Thanks ! I adapted the commit to your comment